packages/help-center/src/hooks/use-opening-coordinates.ts

```diff
@@ -25,8 +25,9 @@ export const calculateOpeningPosition = ( element: HTMLElement ) => {
 	}
 
 	// This takes into account the '?' button's padding and align the close button with the help center icon
-	const computedStyle = window.getComputedStyle( element );
-	const helpCenterPaddingRight = parseInt( computedStyle.paddingRight, 10 );
+	const helpCenterIconPaddingRight =
+		parseInt( element.style.paddingRight, 10 ) ||
+		parseInt( element.getAttribute( 'data-padding-right' ) || '0', 10 );
```

Comment:
You can do `element.dataset.paddingRight` btw. It automatically covnerts kebab case to camel case.